test/github.spec.js

```diff
@@ -0,0 +1,18 @@
+import test from 'ava';
+import sinon from 'sinon';
+import path from 'path';
+import { readGithubToken } from '../lib/github';
+
+const fs = require('fs');
+
+test('readGithubToken returns appropriate token', t => {
+  sinon
+    .stub(fs, 'readFileSync')
+    .returns(
+      fs.readFileSync(path.join(__dirname, '/fixtures/git-credentials-1.txt')),
```

Comment:
Unfortunately this test doesn't work for me. I'm getting:

      TypeError: Cannot read property 'sections' of undefined
          at new SourceMapConsumer (/home/lukasz/github/pwr/node_modules/source-map-support/node_modules/source-map/lib/source-map-consumer.js:20:20)
          at mapSourcePosition (/home/lukasz/github/pwr/node_modules/source-map-support/source-map-support.js:190:14)
          at wrapCallSite (/home/lukasz/github/pwr/node_modules/source-map-support/source-map-support.js:358:20)
          at /home/lukasz/github/pwr/node_modules/source-map-support/source-map-support.js:395:26

stack. Do you have better idea how to test `readGithubToken` function ?

Reply:
Can you please check this? https://github.com/jesusprubio/pwr/pull/11#discussion_r267653810

Reply:
I've did it, now I'm getting [another error](https://travis-ci.org/jesusprubio/pwr/jobs/509329198):

      Error: Internal runner error: Could not serialize error
          at serializeError (/home/travis/build/jesusprubio/pwr/node_modules/ava/lib/serialize-error.js:143:23)
          at runner.on.error (/home/travis/build/jesusprubio/pwr/node_modules/ava/lib/worker/subprocess.js:59:42)
          at Promise.all.staticListeners.map (/home/travis/build/jesusprubio/pwr/node_modules/emittery/index.js:69:13)

I've tried to use `console.log` to debug tests, but I'm getting no output from it. I've tried to add it in `/home/lukasz/github/pwr/node_modules/ava/lib/serialize-error.js` file.